PPC64le + GCC 11.2.1: error: conversion from 'uint64_t' {aka 'long unsigned int'} to 'double' may change value

Some casting issue maybe?


```
[  8%] Building CXX object CMakeFiles/hwy.dir/hwy/nanobenchmark.cc.o
/usr/bin/g++  -I/builddir/build/BUILD/highway-0.14.1 -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -m64 -mcpu=power8 -mtune=power8 -fasynchronous-unwind-tables -fstack-clash-protection -O2 -g -DNDEBUG -fPIC -Wno-builtin-macro-redefined -D__DATE__=\"redacted\" -D__TIMESTAMP__=\"redacted\" -D__TIME__=\"redacted\" -fmerge-all-constants -Werror -Wall -Wextra -Wconversion -Wsign-conversion -Wvla -Wnon-virtual-dtor -fmath-errno -fno-exceptions -std=c++11 -MD -MT CMakeFiles/hwy.dir/hwy/nanobenchmark.cc.o -MF CMakeFiles/hwy.dir/hwy/nanobenchmark.cc.o.d -o CMakeFiles/hwy.dir/hwy/nanobenchmark.cc.o -c /builddir/build/BUILD/highway-0.14.1/hwy/nanobenchmark.cc
/builddir/build/BUILD/highway-0.14.1/hwy/nanobenchmark.cc: In function 'double hwy::platform::InvariantTicksPerSecond()':
/builddir/build/BUILD/highway-0.14.1/hwy/nanobenchmark.cc:403:33: error: conversion from 'uint64_t' {aka 'long unsigned int'} to 'double' may change value [-Werror=conversion]
  403 |   return __ppc_get_timebase_freq();
      |          ~~~~~~~~~~~~~~~~~~~~~~~^~
```

Full log at https://koji.fedoraproject.org/koji/taskinfo?taskID=74091486

--- a/hwy/nanobenchmark.cc
+++ b/hwy/nanobenchmark.cc
@@ -400,7 +400,7 @@ double NominalClockRate() {
 
 double InvariantTicksPerSecond() {
 #if HWY_ARCH_PPC
-  return __ppc_get_timebase_freq();
+  return double(__ppc_get_timebase_freq());
 #elif HWY_ARCH_X86
   // We assume the TSC is invariant; it is on all recent Intel/AMD CPUs.
   return NominalClockRate();

